src/agent/include/unix_daemon.hpp

```diff
@@ -1,3 +1,17 @@
 #pragma once
 
-#include <process_options.hpp>
+#include <string>
+#include <unistd.h>
+#include <vector>
+
+class AgentDaemon
+{
+public:
+    static int Daemonize();
+    static bool IsAlreadyRunning();
+    static void Stop();
+    static std::string GetStatus();
+
+private:
+    static std::vector<pid_t> GetPidsByName(const std::string_view process_name);
```

Comment:
There's a missing include for string view due to this function.

Reply:
Done